shared/src/main/scala/com/cibo/evilplot/plot/renderers/PathRenderer.scala

```diff
@@ -123,6 +125,40 @@ object PathRenderer {
       lineStyle
     )
 
+  /**
+    * Render line with colors based on a third, continuous variable.
+    * @param depths The depths for each line segment.
+    * @param coloring The coloring to use.
+    * @param strokeWidth The width of the path.
+    * @param lineStyle The style of the path (dashed, solid, etc).
+    */
+  def depthColor(
```

Comment:
this is cool, but a little nonstandard? I was thinking more along the lines of being able to supply a coloring to color the different series in a multiline plot, because as it currently stands people have to manually supply the colors for different series themselves.

I think I understand what you were saying about that potentially not jiving with the `LinePlot` constructor/PathRenderer signature as currently written, we might have to do some more thinking about it...

Reply:
@elijahsummers and I talked about it, as it currently stands there isn't a specific "multiline" plot. If we were to add one that should be a stand-alone class itself, correct? currently, xy plot can only render a single `Seq[Double]` so it wouldn't be possible to create an `apply` method for LinePlot that takes in multiple lines. I guess I could add an `apply` that overlays a bunch of lines within a `LinePlot` apply, but that becomes messy.